apps/src/dance/lab2/views/GenerateDancer.tsx

```diff
@@ -182,11 +259,30 @@ const GenerateDancer: React.FunctionComponent<DancerGenerateProps> = ({
     }
     variantHistory.current = newVariantsHistory;
 
+    // Special case: for the creature-attire-mood-style-04 adlib only,
+    // move mood from choices to choicesExtra, and use a unique path.
+    // This is because the style option is not used in retrieving the
+    // head image, and is instead used to retrieve the body.
+    let choicesToSave;
+    let choicesExtraToSave;
+    let pathToSave;
+    if (adlibOption === 'creature-attire-mood-style-04') {
+      pathToSave = 'creature-attire-mood-04';
+      choicesToSave = choices?.slice(0, -1);
```

Comment:
This whole block feels a bit hard-coded, especially considering it's likely to become the new convention going forwards. No major objections right now but worth thinking about more I think.

Reply:
Oh, completely agreed.  I explored modifying the `Adlib` system but it seemed the wrong place to push the complexity.  I also explored bigger modifications to the `AdlibType` format, but given that we have a bunch of existing adlib definitions in both code and many levels, I didn't want to make a big change there either.  

I had hoped to make this a more generic system feature, but given that this is a very specific and very late-breaking change, I think this hardcoding is probably the right balance for now.

Reply:
As discussed, will consider for a follow-up!